### PR TITLE
Update CircleListViewRenderer

### PR DESCRIPTION
**Pull request by myroot**
### Description of Change ###
Update CircleListViewRenderer to reuse X.F.ListViewRenderer

### Bugs Fixed ###
None

### API Changes ###
Removed : 
Tizen.Wearable.CircularUI.Forms.Renderer.CircleListView class

### Behavioral Changes ###
None

### CODE REVIEW

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleListViewRenderer.cs [hunk 1]
```diff
@@ -14,243 +14,35 @@
...
-using System.Collections.Specialized;
 using Xamarin.Forms;
-using Xamarin.Forms.Internals;
 using Xamarin.Forms.Platform.Tizen;
 using XForms = Xamarin.Forms.Forms;
 
 using CCircularListView = global::Tizen.Wearable.CircularUI.Forms.CircleListView;
 
-[assembly: ExportRenderer(typeof(Tizen.Wearable.CircularUI.Forms.CircleListView), typeof(Tizen.Wearable.CircularUI.Forms.Renderer.CircleListViewRenderer))]
+[assembly: ExportRenderer(typeof(CCircularListView), typeof(Tizen.Wearable.CircularUI.Forms.Renderer.CircleListViewRenderer))]
```
Comment (rookiejava) [inline src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleListViewRenderer.cs:23]: CCircleListView or just CircularListView is better

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleListViewRenderer.cs [hunk 2]
```diff
@@ -14,243 +14,35 @@
...
-                _selectedItemChanging--;
-            }
-            else
+            if (!Element.BarColor.IsDefault)
             {
-                int index = Element.TemplatedItems.IndexOf(context.Cell);
-                _selectedItemChanging++;
-                Element.NotifyRowTapped(index);
-                _selectedItemChanging--;
+                (Control as WatchListView).CircleGenList.VerticalScrollBarColor = Element.BarColor.ToNative();
```
Comment (rookiejava) [inline src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleListViewRenderer.cs:45]: (Control as WatchListView).?

Comment (myroot) [reply src/Tizen.Wearable.CircularUI.Forms.Renderer/CircleListViewRenderer.cs:45]: can't because assign statement

**Comment by rookiejava** [comment]
Should remove `ListViewCache` as well.

**Pull request merged**